docs/en/docs/lakehouse/multi-catalog/paimon.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
need add this new doc to `docs/sidebars.json`

Reply:
> 

done